free-1.md

```diff
@@ -57,12 +99,39 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_Most would probably be language design decision based on the semantics
+of building the language. Hopefully once the strcture of the language
+is there, the other language design decisions would be implemented
+based on what is appropriate for that structure. Because all of it is
+so connected in hopes of delivering one product, all maintenance and
+new features would have to interface well with all the old features._
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+_This project has a rather large scope. Even starting with the best way to
+get all the tasks one person needs to get done such that the most important
+tasks are done first, the implementer needs to figure out how they want the
+language to look to the user. Once that's built, it's about how to add features
+to the language, which widens the scope immensly. Including multiple people
+into the to do list, figuring out how to optimize time based on a
+scheudle, and so many other features could be added based on what others
+think would be the most useful._
```

Comment:
I agree. This is a really ambitious project. There are many different apps that try to make to-do lists easy to work with and I think everybody has their own preferences and workflow. Even without the additional factor of coordinating multiple people, it would be hard to create a system that's intuitive to a lot of people.